src/main/java/org/isf/medicalinventory/service/MedicalInventoryIoOperation.java

```diff
@@ -198,4 +202,32 @@ public MedicalInventory getInventoryById(Integer inventoryId) throws OHServiceEx
 		}
 		return null;
 	}
+
+	/**
+	 * Marks an inventory as deleted by changing its status.
+	 * @param inventoryId - the ID of the inventory to delete.
+	 * @throws OHServiceException if an error occurs during the operation.
+	 */
+	public void deleteInventory(int inventoryId) throws OHServiceException {
+		List<OHExceptionMessage> errors = new ArrayList<>();
```

Comment:
Why use an List?   The way I read the code only one of 3 errors can be generated at any time.  Why not just throw each one as it happens?

Reply:
You hav right, I have applyed the changes as you suggested.